provider/aci.go

```diff
@@ -728,6 +738,26 @@ func (p *ACIProvider) CreatePod(ctx context.Context, pod *v1.Pod) error {
 	return p.createContainerGroup(ctx, pod.Namespace, pod.Name, &containerGroup)
 }
 
+// Set the Container Group Priority Property
+// Accepted Values : Regular, Spot
```

Comment:
Since the priority is not an explicit parameter to this function, this comment should explain how priority is passed in, if we already mention what the accepted values are :) 

Additionally, as far as I can tell from the code, it is also allowed not to specify it at all, so I'd recommend mentioning that too.